app/views/dashboard/assessmentDetails.scala.html

```diff
@@ -57,53 +82,48 @@
                     </tr>
 
                     @list.map {  case (url, assessment) =>
-                        <tr class="govuk-table__row">
-                            <td class="govuk-table__cell">
-                                @assessment.effectiveDate.map(Formatters.formatDate).getOrElse("")
-                            </td>
+                    <tr class="govuk-table__row">
+
+                        <td class="govuk-table__cell">@effectiveDate(assessment)</td>
+
+                        @assessment.rateableValue match {
```

Comment:
un thus template I see the same kind of problem repeatedly:

```
 @assessment.rateableValue match {
                            case Some(rv) => {
                                <td class="govuk-table__cell">£@rv</td>
                            }
                            case None => {
                                <td class="govuk-table__cell">@messages("assessments.na")</td>
                            }
                        }
```

regardless of whether the value is Some or None, you'd always want to output the  `<td class="govuk-table__cell">` and the `</td>`

Why repeat the element?

this sort of code is what you want I think:

```
<td class="govuk-table__cell">
  @assessment.rateableValue.fold(messages("assessments.na"))( rv => s"£$rv" )
</td>
```

Reply:
true - it's existing code, but I will tidy that up